keyboards/primekb/meridian/v1/config.h

```diff
@@ -0,0 +1,6 @@
+//Data Transfer Time for WS2812C LEDs as per datasheet spec.
```

Comment:
```suggestion
/*
Copyright 2020 Holten Campbell
This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 2 of the License, or
(at your option) any later version.
This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.
You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
*/
#pragma once
//Data Transfer Time for WS2812C LEDs as per datasheet spec.
```